api/app_sph_lms/models.py

```diff
@@ -1,436 +1,436 @@
+    tag = models.ForeignKey(Tag, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    class Meta:
+        unique_together = ('course', 'tag')
+        verbose_name = "CourseTag"
+        verbose_name_plural = "CourseTag"
+        db_table = "app_sph_lms_course_tags"
+
+    def __str__(self):
+        return "Course: " + str(self.course) + " | " + "Tag: " + str(self.tag)
+
+
+class Lesson(models.Model):
+    course = models.ForeignKey(
+            Course,
+            on_delete=models.CASCADE,
+            related_name="lessons"
+        )
+    title = models.CharField(
+            max_length=255,
+            validators=[MinLengthValidator(3)]
+        )
+    link = models.URLField(max_length=600)
+    description = models.TextField(
+            max_length=65000,
+            null=True,
+            validators=[MinLengthValidator(5)]
+        )
+    is_active = models.BooleanField(default=True)
+    order = models.IntegerField(default=0)
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    def __str__(self):
+        return self.title
+
+
+class CourseTrainee(models.Model):
+    trainee = models.ForeignKey(
+            Trainee, on_delete=models.CASCADE,
+            related_name="trainee_detail"
+        )
+    course = models.ForeignKey(
+            Course,
+            on_delete=models.CASCADE,
+            related_name="course_detail"
+        )
+    is_active = models.BooleanField(default=1)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        unique_together = ('trainee', 'course')
+        verbose_name = "CourseTrainee"
+        verbose_name_plural = "CourseTrainees"
+        db_table = "app_sph_lms_course_trainees"
+
+    def __str__(self):
+        return (
+            "Trainee: "
+            + str(self.trainee)
+            + " | "
+            + "Course: "
+            + str(self.course)
+        )
+
+
+class LearningPath(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[MinLengthValidator(3)]
+    )
+    author = models.ForeignKey(
+            User,
+            on_delete=models.CASCADE,
+            related_name='learning_path'
+        )
+    description = models.TextField(
+            max_length=65000,
+            null=True,
+            validators=[MinLengthValidator(5)]
+        )
+    image = models.CharField(max_length=255, null=True)
+    is_active = models.BooleanField(default=True)
+    courses = models.ManyToManyField(
+        Course,
+        related_name="learning_path",
+        through='LearningPathCourse'
+        )
+    category = models.ManyToManyField(Category, related_name="category")
+    trainee = models.ManyToManyField(User, related_name='trainee')
```

Comment:
```suggestion
    trainee = models.ManyToManyField(User, related_name='enrolled_learning_paths')
```

Please change the `related_name`, since `related_name` is the name of the reverse relation, `enrolled_learning_paths `makes more sense to get from the user table.

so when we get all learning paths that trainee enrolled from users table, it would look something like user.enrolled_learning_paths.all()